src/Symfony/Component/Config/Tests/Definition/ScalarNodeTest.php

```diff
@@ -22,7 +22,7 @@ class ScalarNodeTest extends TestCase
     /**
      * @dataProvider getValidValues
      */
-    public function testNormalize($value)
+    public function testNormalize(mixed $value)
```

Comment:
I would also suggest to skip adding types to test cases: more merge conflicts for little to no benefit

Reply:
My work on this PR has identified one lying `@param` annotation in the tests. That's a benefit, I'd say. I'm okay with removing the `mixed` types here, but may I keep the other union types?

Reply:
works for me